### DISCUSSION TITLE
Is there a way to add completion in the cmdline mode?

### DISCUSSION THREAD

**Discussion by alankritjoshi** [saghen/blink.cmp#630] [Q&A]
I had been using this hack in the nvim-cmp so was interested in the equivalent here.

```
local cmp = require("cmp")
      cmp.setup.cmdline(":", {
        mapping = cmp.mapping.preset.cmdline(),
        sources = cmp.config.sources({
          { name = "path" },
          { name = "buffer" },
        }),
      })
```

**Comment by saghen** [comment]
It's enabled by default on the `main` branch, it's not out in a release yet

**Comment by fabiomcosta** [comment]
This is on by default as of 0.8: https://github.com/Saghen/blink.cmp/releases/tag/v0.8.0

**Comment by minusfive** [comment]
If you're using `LazyVim` it's disabled there by default (because it used to be unstable). I re-enabled it with:

```lua
      sources = {
        -- Force re-enable commandline completion
        cmdline = require("blink.cmp.config.sources").default.cmdline,
      },

```
